### PR TITLE
fix(footer-#49): display dynamic date, ensure all link are functional

### LINKED ISSUE #49: Footer
Content and functionality issues.

**Description:**
1. Static Date/Time displayed.
2. Dead links in social icons, teams, and documentation.

**Expected Behavior:**
- Display dynamic Date/Time.
- Ensure all links are functional.

**ETA**
This issue is expected to be completed within 2 days of assignment. Failure to create a pull request (or draft pull request) within the expected time will lead to being unassigned.

### PR DESCRIPTION
**Pull request by portableDD**
## Description

1. Created a new footer.ts file to create all my arrays.
2. updated the footer.tsx to map through the arrays

## Related Issue(s)

Closes #49
## Checklist:

- [x] Read the [contributing docs](../CONTRIBUTING.md) (if this is your first contribution)
- [x] Verified this is not a duplicate of any existing pull request

- [x] Performed a self-review of the code
- [ ] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Linked the issues which this PR resolves
- [ ] Documented the changes in code (API docs will be generated automatically)

### CODE REVIEW

**Comment by portableDD** [comment]
Please review @Abeeujah

**Review by princeibs — CHANGES_REQUESTED** [review]
Hi @portableDD, can you fix issue causing build to fail?

**Comment by portableDD** [comment]
@princeibs the issues has been fixed

File: app/components/Footer.ts
Comment (princeibs) [inline app/components/Footer.ts:1]: Please add the correct links.

Twitter: `https://twitter.com/auto_swappr`
Telegram: `https://t.co/yFNm0juRkX`
Github: `https://github.com/BlockheaderWeb3-Community/autoswappr-dapp`
Documentation: `/documentation`

**Review by princeibs — CHANGES_REQUESTED** [review]

**Review by Utilitycoder — APPROVED** [review]

**Pull request merged**